src/pipe.ts

```diff
@@ -138,12 +142,31 @@ export function createPipe<Ctx extends PipeCtx = PipeCtx<any>>({
     middlewareMap[`${createName}`] = fn || defaultMiddleware;
 
     const tt = req ? (req as any).saga : takeEvery;
+
     function* curSaga(): SagaIterator<void> {
       yield tt(`${action}`, onApi);
     }
     sagas[`${createName}`] = curSaga;
+
     const actionFn = (options?: any) => {
-      const key = encodeBase64(JSON.stringify({ name: createName, options }));
+      const koSort = (opts?: any): object => {
+        if (opts === null || opts === undefined) return {};
+        if (!isObject(opts)) return { opts: opts };
+        return Object.keys(opts)
+          .sort()
+          .reduce((res: any, key: any) => {
+            res[`${key}`] = opts[key];
+            if (opts[key] && typeof opts[key] === 'object') {
```

Comment:
Is there a reason why you didn't use `isObject` here?

Reply:
No. I just observed the custom "isObject"  function later.